src/test/java/com/scalyr/integrations/kafka/AddEventsClientTest.java

```diff
@@ -420,14 +423,20 @@ public void testTooLargeAddEventsSkipped() throws Exception {
     AddEventsClient addEventsClient = new AddEventsClient(scalyrUrl, API_KEY_VALUE, ADD_EVENTS_TIMEOUT_MS, ADD_EVENTS_RETRY_DELAY_MS, compressor);
     List<Event> events = createTestEvents(numEvents, numServers, numLogFiles, numParsers);
     events.forEach(event -> event.setMessage(largeMsg));
-    addEventsClient.log(events);
+
+    assertEquals(0, server.getRequestCount());
+
+    addEventsResponse = addEventsClient.log(events).get(5, TimeUnit.SECONDS);
```

Comment:
Remove assignment to `addEventsResponse` since we don't do anything with it.  i.e. Just `addEventsClient.log(events).get(5, TimeUnit.SECONDS)`.

Reply:
oops, I actually wanted to assert on the response as well - to ensure mock server is indeed hit. Will add that change.

Reply:
Addressed in c5f9344e133c9c1ccf14755841d6cae1db4adabb.